### PR TITLE
New queue-ing system

### PR DESCRIPTION
**Pull request by micahcarroll**
- Updated the queuing system so that there is no single/multi anymore, and runs faster
- Set up test that runs pretty quickly to test current implementation
- Other code cleaning

~NOTE: the queueing system is a little stochastic, in that sometimes we will create a couple more trajectories than necessary, and sometimes we will create a couple less trajectories than necessary.~

### CODE REVIEW

File: influence_benchmark/environment_vectorized/environment_vectorized.py [hunk 1]
```diff
@@ -181,6 +176,13 @@ def generate_trajectories(
                         "transition_probs": env.current_state.transition_probs,
                     }
                 )
+
+            # NOTE: Marcus does moving this stuff here seem fine to you? I think it should be?
```
Comment (micahcarroll) [inline influence_benchmark/environment_vectorized/environment_vectorized.py:180]: Highlighting this comment

**Comment by micahcarroll** [comment]
As an update to the original description, I eventually made the queueing system reliably create the right amount of trajectories. That being said, it is still stochastic and breaks seeding, because we have multiple queues, and which GPU is allocated which trajectory depends on which GPU frees up when

**Comment by carolius** [comment]
Maybe we don't care that much about seeding anyway?

**Comment by micahcarroll** [comment]
Yeah I think we can't have both seeding and an efficient queue, so fuck it

File: influence_benchmark/environment_vectorized/environment_vectorized.py (hunk 1 above)
Comment (carolius) [reply influence_benchmark/environment_vectorized/environment_vectorized.py:186]: Yeah it should be

Comment (micahcarroll) [inline influence_benchmark/environment_vectorized/environment_vectorized.py:186]:
```suggestion

```

**Pull request merged**